enlighten/Plugins/PluginController.py

```diff
@@ -499,7 +508,7 @@ def do_post_disconnect(self):
     # plus set the "process" button to the opposite state
     def enabled_callback(self):
         self.enabled = self.cb_enabled.isChecked()
-        if self.button_process is not None:
+        if self.button_process is not None:     # MZ: when would that be none?
```

Comment:
Trying to make the habit of using msgboxes to expose things, you could add an else with an `msgbox("unexpected value button_process is not None")`.

The rule for the msgbox's is that the message should be short enough and distinctive enough that someone non-technical can remember it an hour after seeing it.

If two msgboxes are very similar, then I like to vary the wording
"file read error" == "corrupted file" == "fatal error: FILE IO"

We can always go back and prove out a question like "when would this be none", so this is a way to expedite that when we don't have time... (most of the time).